hypershift-operator/controllers/sharedingress/testdata/zz_fixture_TestGenerateConfig_When_there_s_two_HostedClusters_with_all_their_Routes_and_SVCs_it_should_generate_the_config_with_frontends_and_backends_for_both.cfg

```diff
@@ -19,9 +19,14 @@ defaults
 frontend dataplane-kas-svc
   bind :::6443 v4v6 accept-proxy
   tcp-request inspect-delay 5s
+
+  # fc_pp_tlv(0x20) extracts and decodes the custom tlv proxy header sent from the data-pane HAProxy which contains the clusterID.
+  tcp-request content set-var(sess.cluster_id) fc_pp_tlv(0x20)
   tcp-request content accept if { req_ssl_hello_type 1 }
-  acl is_test-hc1-kube-apiserver dst 4.4.4.4
-  acl is_test-hc2-kube-apiserver dst 4.4.4.4
+
+  log-format "%{+Q}o\ custom_proxy_tlv_header = %[var(sess.cluster_id)], %ci:%cp -> %fi:%fp [%t] %ft %b/%s %Tw/%Tc/%Tt %B %ts %ac/%fc/%bc/%sc/%rc %sq/%bq"
```

Comment:
can we add a comment with logging example or a link to a source?

Reply:
added an example output with a link explaining the different logging vars.